tests/testthat/test-expect-no-condition.R

```diff
@@ -1,3 +1,18 @@
+
+test_that("expect_no_* conditions behave as expected", {
+
+  # base R
+  expect_snapshot_failure(expect_no_error(stop("error")))
```

Comment:
I switched these to use snapshot tests which made me re-read and re-consider the failure text. What do you think of the new output?

Reply:
I like it!